backend/src/main/java/com/huellapositiva/domain/model/valueobjects/ProposalStatus.java

```diff
@@ -8,7 +8,7 @@
     REVIEW_PENDING (1),
     CHANGES_REQUESTED (2),
     PUBLISHED (3),
-    UNPUBLISHED (4),
+    CLOSED (4),
```

Comment:
If we would have existing data stored in the DB, this change would break its consistency.

As a general rule, for extending and versioning existing contracts it is safer to add/remove elements but to not modify them. Does CLOSED and UNPUBLISHED mean semantically the same? If yes, go on, if not, then create a new value for CLOSED.

ENROLLMENT_CLOSED would be more meaninful